### ISSUE TITLE
Allow running with multiple configurations

### ISSUE THREAD

**Issue by sarayourfriend** [preactjs/compressed-size-action#81]
Our application (https://github.com/WordPress/openverse-frontend) generates two distinct bundle groups during the build process, a "modern" and "universal" bundle (this is a concept we inherit by using Nuxt). We'd like to have separate compressed size checks for each of the two build types, however, when you try to run the compressed-size-check action twice in one PR, it overwrites the comment.

You can observe the issue in this PR: https://github.com/WordPress/openverse-frontend/pull/1378

If you check the edit history of the bot's comment, you can see that it overwrote itself.

Would y'all accept a PR to add an optional "key"/"id"/"comment-key" (names are hard) input that could be used to specify the comment to update? 

Essentially my idea would be to update this conditional to additionally check for the new input in the body: https://github.com/preactjs/compressed-size-action/blob/master/src/index.js#L202-L205

For example, instead of just `compressed-size-check` being in the body, it could be `compressed-size-check::${key}` or something along those lines.

Thanks!

**Comment by developit** [comment]
A comment key configuration would be great!

**Comment by sarayourfriend** [comment]
Hi @developit, I'm going to take a try implementing this today.

**Comment by sarayourfriend** [comment]
I've reopened the PR. If you can help wrangle a review @slorber I am happy to make changes or otherwise make it possible to get that change committed.

**Comment by rschristian** [comment]
Supported via #82

Thanks again for the patience!

**Comment by slorber** [comment]
Awesome thanks

Do you plan to do a release soon? Or is there a way to use a "canary" action until then or something? 😄

**Comment by rschristian** [comment]
Will release soon-ish

**Comment by sarayourfriend** [comment]
> Do you plan to do a release soon? Or is there a way to use a "canary" action until then or something? 😄

@slorber you can ping actions to any valid git ref on the main branch of the action repository, not just tags/releases. [HEAD is at `4a35a533ce71b6edb72f2049f4c13d0611d17607` as of writing](https://github.com/preactjs/compressed-size-action/commit/4a35a533ce71b6edb72f2049f4c13d0611d17607), so something like this would do the trick:

```yaml
name: Compressed Size
on: [pull_request]
jobs:
  modern-bundle-size-check:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - uses: preactjs/compressed-size-action@4a35a533ce71b6edb72f2049f4c13d0611d17607
      with:
        build-script: "build:modern"
        comment-key: modern
```

**Comment by slorber** [comment]
Thanks! Will try that asap.

**Comment by rschristian** [comment]
Just released as 2.7.0/v2
